.travis.yml

```diff
@@ -2,3 +2,4 @@ language: ruby
 rvm:
   - 2.1.0
   - 2.1.1
+  - 2.3.1
```

Comment:
While we are here, why don't we please clean this up a bit.

```yml
language: ruby
rvm:
- 2.1
- 2.2
- 2.3
matrix:
  allow_failures:
  - rvm: ruby-head
```

I [validated](http://lint.travis-ci.org/) this YAML with Travis WebLint.